src/state/dimension.js

```diff
@@ -161,6 +161,7 @@ export const getDroppableDimension = ({
       },
       page: {
         withoutMargin: getFragment(containerRectWithWindowScroll),
+        withMargin: getFragment(getWithSpacing(containerRectWithWindowScroll, margin)),
```

Comment:
This is the margin of the droppable and not the container - perhaps the container margin should be an extra argument?